crates/glaredb/src/commands.rs

```diff
@@ -55,6 +55,11 @@ trait RunCommand {
 
 impl RunCommand for LocalArgs {
     fn run(self) -> Result<()> {
+        if self.debug_tokio {
+            eprintln!("> Starting tokio debugger");
+            console_subscriber::init();
+        }
```

Comment:
is there an easy way to ifdef this out in release mode?

Reply:
I'll do that!

Reply:
Updated!